src/locales/locale-en.json

```diff
@@ -50,7 +36,11 @@
   },
   "FORM": {
```

Comment:
I would name this `VALIDATION` personally

Reply:
makes sense, will do